DSTClusterConfig/Master/leveldataoverride.lua

```diff
@@ -112,6 +173,12 @@ return {
   },
   required_prefabs={ "multiplayer_portal" },
   required_setpieces={ "Sculptures_1", "Maxwell5" },
+  settings_desc="The standard Don't Starve experience.",
+  settings_id="SURVIVAL_TOGETHER",
+  settings_name="Standard Forest",
   substitutes={  },
-  version=4 
-}
+  version=4,
+  worldgen_desc="The standard Don't Starve experience.",
+  worldgen_id="SURVIVAL_TOGETHER",
+  worldgen_name="Standard Forest" 
```

Comment:
Why the duplicated values? `worldgen_` and `settings_` `desc`, `id` and `name` have the same values, are both of them necessary (or autogenerated)? 🤔  I noticed this both in Caves and Master config. 

Also, if these are the new defaults, can we remove `desc`, `id` and `name` from the top of both files? 💡

Reply:
These are the new defaults actually.

As for why, earlier this year DST started using separate configs for world generation (only during generation) and world settings (anything after generation). I guess so that people can have separate presets for each part.

Please see [March 2021 QOL Update](https://dontstarve.fandom.com/wiki/Don%27t_Starve_Together/Version_History#March_11.2C_2021_-_March_2021_QOL_Update).
> Hey everybody.
>
> We're back with another QOL update, this update contains some changes to world options, they have been split into two categories, world settings (things affect the world after it has been generated), and world generation options (things that affect world generation). By default, saving and loading presets will work as before, or can choose to unlink the world settings and generation options and manage their presets separately. If you host a dedicated server, the method for applying world options is unchanged and will continue to work just fine.

In case you are wondering then everything here is auto-generated with my latest DST client. I made no change to these configs myself.

Reply:
Thanks for clarifying! 💯